WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -46,8 +49,13 @@ public void onErrorResponse(VolleyError volleyError) {
             }
         };
 
+        Map<String, String> params = new HashMap<>();
+        params.put("number", Integer.toString(PeopleUtils.FETCH_USERS_LIMIT));
+        params.put("offset", Integer.toString(offset));
+        params.put("order_by", "display_name");
```

Comment:
Yes! This is very weird indeed. In the response we have `name` as the display name, however if you use the [console](https://developer.wordpress.com/docs/api/console/) to `order_by` the `name` it doesn't order it the way we want. You actually need to use `display_name` even though it's not in the response at all and what's more is that the default order is descending. 😕 

We should actually ping @ebinnion in case this is something we want to improve on the API side.